sorting/count_sort/count_sort.py

```diff
@@ -0,0 +1,43 @@
+"""
+Logic to handle negative numbers in counting sort was referred from the below link:
+https://www.ripublication.com/ijaer18/ijaerv13n1_28.pdf
+"""
+def counting_sort(L):
+    if not L:
+        return L
+        
+    largest = max(L)
+    min_value = min(L)
+    offset = -min_value if min_value < 0 else 0
+    n = len(L)
+    
+    # Initialize the frequency array.
+    frequencyArray = [0]*(largest+1+offset)
+
+    # Count the frequency of each element in the array.
+    for i in range(n):
+        # Shift the element by the offset which ensures negative numbers are shifted to non-negative indices in the frequencyArray.

```

Comment:
comment line length too long. make it a two-line comment.
https://google.github.io/styleguide/pyguide.html#32-line-length

Reply:
wow, I learned how to comment properly.